pcsx2/gui/AppMain.cpp

```diff
@@ -536,6 +538,27 @@ void DoFmvSwitch()
 	paused_core.AllowResume();
 }
 
+void Do43Switch(int on)
+{
+	ScopedCoreThreadPause paused_core(new SysExecEvent_SaveSinglePlugin(PluginId_GS));
+	AspectRatioType& art = g_Conf->GSWindow.AspectRatio;
+
+	if (on == 1 && art != AspectRatio_4_3) {
+		prevAR = art;
+		art = AspectRatio_4_3;
+	}
+
+	if (on == 0) {
+		art = prevAR;
+	}
+
+	if (GSFrame* gsFrame = wxGetApp().GetGsFramePtr())
+		if (GSPanel* woot = gsFrame->GetViewport())
```

Comment:
That's code I copied from the elsewhere in PCSX2, it's actually used elsewhere in the same file too. That's why I left the name alone, though I do agree a more informative name would be better.